tests/python/gaia-ui-tests/gaiatest/tests/functional/email/test_IMAP_email_notification.py

```diff
@@ -60,8 +60,11 @@ def test_IMAP_email_notification(self):
 
         # Assert there is one notification and is listed in notifications-container
         notifications = utility_tray.notifications
-        self.assertEqual(1, len(notifications), 'Expected one notification.')
-        email = notifications[0].tap_notification()
```

Comment:
As we use the for loop in 2 tests, I'd factorize in the page class:
```python
     def get_notifications(self, for_app=None):
         return [Notification(self.marionette, notification)
                 for notification in self.marionette.find_elements(*self._desktop_notifications_locator) if for_app is None or for_app in notification.manifest]
```

So we could use in the tests: 
```python
notifications = utility_tray.notifications(for_app='email')
self.assertEqual(1, len(notifications), 'Expected one email notification.')
email = notifications.tap_notification()
```